packages/sdk/shell/src/panels/IdentityPanel/steps/ProfileForm.tsx

```diff
@@ -10,11 +10,19 @@ import { type Identity } from '@dxos/react-client/halo';
 import { useTranslation } from '@dxos/react-ui';
 import { hexToEmoji, hexToHue } from '@dxos/util';
 
-import { Action, Actions, StepHeading, Input, useClipboardContext, EmojiPicker, HuePicker } from '../../../components';
+import {
+  Action,
+  Actions,
+  StepHeading,
+  Input,
+  useClipboardContext,
+  EmojiPickerBlock,
+  HuePickerBlock,
+} from '../../../components';
 import { type IdentityPanelStepProps } from '../IdentityPanelProps';
 import { type IdentityEvent } from '../identityMachine';
 
-export interface ProfileFormProps extends Omit<IdentityPanelStepProps, 'send'> {
+export interface ProfileFormProps extends Omit<IdentityPanelStepProps, 'send' | 'devices'> {
```

Comment:
is this form still used somewhere?

Reply:
It could be, if hue/icon can be adjusted for devices. Could maybe just remove for now, I’m not sure we’d want to keep much of its current implementation.